### DISCUSSION TITLE
Benchmarks Frontend Source Code

### DISCUSSION THREAD

**Discussion by gotzmann** [TechEmpower/FrameworkBenchmarks#6496] [General]
Could someone point me to the source code of the frontend part of Techempower Benchmarks?

I'm thinking of reusing that great data visualisation framework for another benchmarking project.

It seems that license is not restricting that, or am I missing something?

**Comment by NateBrady23** [comment]
The data visualization is not currently open sourced, but we are working on that with the next iteration.
